addons/safeStart/functions/fnc_missionTimeWarning.sqf

```diff
@@ -17,23 +17,21 @@ TRACE_1("Params",_this);
 
 if !(isServer) exitWith {};
 
-params [];
-
 private _missionLength = getMissionConfigValue [QEGVAR(missionTesting,missionTimeLength),0];
 if (_missionLength isEqualType "") then { _missionLength = parseNumber _missionLength; }; // BWC with old attribute that was string
-private _missionLengthSec = _missionLength * 60;
 
-if (_missionLength == 0) then {
-    private _missionType = getMissionConfigValue QEGVAR(missionTesting,missionType);
-    switch (_missionType) do {
-        case 1: {_missionLength = 65};
-        case 2: {_missionLength = 35};
-        default {if(true) exitWith {TRACE_1("Something is seriously wrong, exiting timer",_missionType)}};
-    };
-};
+// if (_missionLength == 0) then {
```

Comment:
@DanThomas64 - this is a follow-up to https://github.com/BourbonWarfare/POTATO/pull/436/files
so as far as I can tell this hasn't worked in a while because of the nil, 
so I think it makes sense to disable this auto time for now
and just only run when explicitly set?